src/tool_path_planning_server.cpp

```diff
@@ -0,0 +1,65 @@
+#include <noether_ros/srv/plan_tool_path.hpp>
+#include <noether_ros/conversions.h>
+
+#include <noether_tpp/core/tool_path_planner_pipeline.h>
+#include <noether_tpp/plugin_interface.h>
+#include <pcl/io/vtk_lib_io.h>
+#include <rclcpp/rclcpp.hpp>
+#include <yaml-cpp/yaml.h>
```

Comment:
Is there a standard somewhere that you are following to determine the order and grouping of these `#include` statements? Just curious. This isn't the pattern I follow, but it's been long enough that I don't remember precisely where my pattern came from and I'd be open to alternatives.